docs/Quantiles.ipynb

```diff
@@ -0,0 +1,562 @@
+    {
+      "cell_type": "code",
+      "execution_count": 18,
+      "metadata": {
+        "id": "eryzri-4XVbN"
+      },
+      "outputs": [],
+      "source": [
+        "mean_age = df.select(pl.col(\"AGE\")).mean().collect()\n",
+        "candidates = list(range(100))"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 6,
+      "metadata": {},
+      "outputs": [
+        {
+          "name": "stdout",
+          "output_type": "stream",
+          "text": [
+            "Actual Median: 47.0\n",
+            "Differentially Private Median: 46\n"
+          ]
+        }
+      ],
+      "source": [
+        "actual_median = list(df.select(pl.col(\"AGE\").median()).collect())[0][0]\n",
+        "print(f\"Actual Median: {actual_median}\")\n",
+        "\n",
+        "dp_median = list(context.query().select(\n",
+        "    pl.col(\"AGE\").fill_null(mean_age).dp.median(candidates)\n",
+        ").release().collect())[0][0]\n",
+        "print(f\"Differentially Private Median: {dp_median}\")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "SnUJxheXXVbN"
+      },
+      "source": [
+        "Let's find the age at the 30th quantile! To compute it using the quantile method directly, follow these steps:\n",
+        "\n",
+        "1. Select the variable you are interested in. In this case we chose \"AGE\"\n",
+        "2. Fill in null values. We'll impute the values with the mean age.\n",
+        "3. Specify the following parameters in the quantile method:\n",
+        "\n",
+        "    a. Quantile: This is between 0 and 1. We provide 0.30. \n",
+        "    \n",
+        "    b. A range of possible values for your quantiles. This will depend on your specific domain. We know that ages are generally between 0 and 100, we use those as our candidates, so our results will be limited to these values."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 7,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 128
+        },
+        "id": "7dLmSCiDXVbN",
+        "outputId": "2dadf2fd-dab1-4e69-82bb-d538f17d3ed2"
+      },
+      "outputs": [
+        {
+          "data": {
+            "text/html": [
+              "<div><style>\n",
+              ".dataframe > thead > tr,\n",
+              ".dataframe > tbody > tr {\n",
+              "  text-align: right;\n",
+              "  white-space: pre-wrap;\n",
+              "}\n",
+              "</style>\n",
+              "<small>shape: (1, 1)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>AGE</th></tr><tr><td>i64</td></tr></thead><tbody><tr><td>30</td></tr></tbody></table></div>"
+            ],
+            "text/plain": [
+              "shape: (1, 1)\n",
+              "┌─────┐\n",
+              "│ AGE │\n",
+              "│ --- │\n",
+              "│ i64 │\n",
+              "╞═════╡\n",
+              "│ 30  │\n",
```

Comment:
It's a little confusing that by coincidence the age at the 30th percentile turns out to be 30. Perhaps choose a higher or lower number, so this coincidence is less likely, and afterwards repeat in plain language what this result means.